City-of-Helsinki/drupal-helfi-etusivu
# UHF-6647: rss feed content type header fix

# [UHF-6647](https://helsinkisolutionoffice.atlassian.net/browse/UHF-0000)

Rss-feed view's cached response was broken due to missing "contenttype=app/rss+xml header"

## What was done


* Installed patch which fixes the problem
* Added style to views' configuration

## How to install

* Use database from testing environment if you don't want to create the news manually

* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-6647_rss_feed_contenttype_header`
  * `make fresh`
* Run `make drush-cr`
* Make sure news items are indexed to elasticsearch

## How to test

* Open network tab
* Go to `/fi/news/rss` And/Or `/fi/news/react/rss`
* Load the page twice in order to get the cached request
* Check the inspector, you should see rss request response's content-type header, the value should be something like `application/rss+xml; charset=utf-8`
* The feed should look like valid xml
* Check that the feed's ääkköset are working properly.
